lib/verify.js

```diff
@@ -3,9 +3,6 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const
-http = require("http"),
-https = require("https"),
-url = require("url"),
```

Comment:
These and similar changes are newly-discovered lint, apparently uncovered by upgraded linter.